tests/testthat/test-labellers.R

```diff
@@ -20,3 +20,31 @@ test_that("labeller function catches overlap in names", {
     )
   expect_snapshot_error(ggplotGrob(p))
 })
+
+# Visual tests ------------------------------------------------------------
+
+test_that("labeller handles unspecified labels from lookup tables", {
```

Comment:
It might be a bit overkill to do a visual test for a textual change. 
The following is a bit more lightweight test for the behaviour this PR fixes.

```r
df <- data_frame0(am = c(0, 1))

labs <- labeller(am = c("0" = "Automatic", "11" = "Manual"))
expect_equal(labs(df), list(am = "Automatic", "1"))

labs <- labeller(am = c("0" = "Automatic"))
expect_equal(labs(df), list(am = "Automatic", "1"))
```

Reply:
Great suggestion! My first instinct was to test in that manner, but I didn't now how to retrieve the _actual_ applied labels to make an `expected_equal` test!

I'll update the tests! Thanks :)